### PR TITLE
[SYCL][DOC][NFC] Add release notes for November release

### PR DESCRIPTION
**Pull request by vladimirlaz**

### CODE REVIEW

File: sycl/ReleaseNotes.md [hunk 1]
```diff
@@ -1,3 +1,208 @@
+# November'20 release notes
+
+Release notes for commit range c9d50752..5d7e0925
+
+## New features
+
```
Comment (pvchupin) [inline sycl/ReleaseNotes.md:6]: none?

File: sycl/ReleaseNotes.md [hunk 2]
```diff
@@ -1,3 +1,208 @@
...
+  - Fix path to FPGA device selector [ca33f7f7]
+  - Rename LEVEL0 environment variable to LEVEL_ZERO in documents and code
+    comments following source code change [2c3908b4]
+  - Clarify --system-ocl key in GetStartedGuide.md [e31b94e5]
+
+## API/ABI breakages
+  - Implement SYCL_INTEL_device_specific_kernel_queries [24ae95b3]
+  - Implement accessor_properties extension [f7d073d1]
+  - Add ONEAPI::filter_selector that accepts one or more filters for device
+    selection [174fd168]
```
Comment (pvchupin) [inline sycl/ReleaseNotes.md:161]: Are these breakages?

File: sycl/ReleaseNotes.md (hunk 1 above)
Comment (vladimirlaz) [reply sycl/ReleaseNotes.md:26]: Move feature items from improvement section

File: sycl/ReleaseNotes.md [hunk 3]
```diff
@@ -1,3 +1,204 @@
...
+### Documentation
+  - Fixed path to FPGA device selector [ca33f7f7]
+  - Renamed LEVEL0 environment variable to LEVEL_ZERO in documents and code
+    comments following source code change [2c3908b4]
+  - Clarified --system-ocl key in GetStartedGuide.md [e31b94e5]
+
+## API/ABI breakages
+  - Implemented SYCL_INTEL_device_specific_kernel_queries [24ae95b3]
+  - Implemented accessor_properties extension [f7d073d1]
+  - Added ONEAPI::filter_selector that accepts one or more filters for device
```
Comment (s-kanaev) [inline sycl/ReleaseNotes.md:156]: I can't see why this change is listed as ABI break.
`Adding a new exported symbol is considered to be non-breaking change.`.

File: sycl/ReleaseNotes.md [hunk 4]
```diff
@@ -1,3 +1,204 @@
...
+    for group algorithms [97bec247]
+  - Added support for L0 loader validation layer [4c6cda3f]
+  - Added multi-device and multi-platform support for SYCL_DEVICE_ALLOWLIST
+    [dbf31c3c]
+  - Removed two-input sub-group shuffles [ef969c14]
+  - Enabled inspecting values wrapped into private_memory<T> by evaluating
+    `operator()` from GDB [31c23ddc]
+  - Used host shared memory when buffer was allocated for integrated GPU under
+    Level Zero plugin [2ae1bc9e]
+  - Implemented queue::parallel_for() accepting reduction [ffdadc2e]
```
Comment (romanovvlad) [inline sycl/ReleaseNotes.md:74]:
```suggestion
  - Implemented `queue::parallel_for()` accepting reduction [ffdadc2e]
```

File: sycl/ReleaseNotes.md [hunk 5]
```diff
@@ -1,3 +1,204 @@
...
+  - Fixed optimization disabling option for gen to use -cl-opt-disable
+    [ba4e567fe]
+  - Emulated "funnel shift left" which was not supported in the OpenCL
+    ExtInst set on SPIRV translator side [97d7eec5]
+  - Fixed build issue when TBB and libstdc++ 10.X were used caused by including
+    std C++ headers in integration header file [63369132]
+  - Fixed processing for partial link step with static archives by passing linker
+    specific arguments there [3ab8cc82]
+  - Enabled -reuse-exe support for Windows [43f2d4ba]
+  - Fixed missing dependency file when compiling for -fintelfpga and using a named
```
Comment (romanovvlad) [inline sycl/ReleaseNotes.md:119]:
```suggestion
  - Enabled `-reuse-exe` support for Windows [43f2d4ba]
  - Fixed missing dependency file when compiling for `-fintelfpga` and using a named
```

File: sycl/ReleaseNotes.md (hunk 4 above)
Comment (romanovvlad) [reply sycl/ReleaseNotes.md:74]: Please, apply to the whole patch.

File: sycl/ReleaseNotes.md [hunk 6]
```diff
@@ -1,3 +1,204 @@
...
+    dependency output file [df5f1ab67]
+  - Set default work group size to {1, 1, 1} to fix out-of-memory crashes on
```
Comment (romanovvlad) [inline sycl/ReleaseNotes.md:121]: Shouldn't this be in the SYCL library section?

File: sycl/ReleaseNotes.md [hunk 7]
```diff
@@ -1,3 +1,204 @@
...
+  - Fixed circular dependency between the device_impl and the platform_impl
+    causing handler leak [255f304f]
+  - Fixed work-group size selection in reductions [2ae49f57e]
+  - Fixed compilation errors when built with --std=c++20 [ecd0adbb]
+  - Fixed treating all allocations of host pointers as read only causing double
+    free issue [8b5506255]
+  - Fixed a problem in Level Zero plugin with kernels and programs destruction
+    while they can be used [b9bf9f5f]
+  - Fixed wrong exception raised by ALLOWLIST mechanism [d81081f7]
+  - Supported GroupBroadcast with 32-bit id to cover broadcast algorithm with
```
Comment (romanovvlad) [inline sycl/ReleaseNotes.md:142]: Doesn't it sound like an improvement?

File: sycl/ReleaseNotes.md (hunk 2 above)
Comment (vladimirlaz) [reply sycl/ReleaseNotes.md]: Only accessor class change is breaking.

**Comment by bader** [comment]
@pvchupin, ping.

**Review by kbobrovs — APPROVED** [review]
LGTM

**Review by pvchupin — APPROVED** [review]

**Pull request merged**